routes/v1.js

```diff
@@ -163,18 +175,19 @@ api.declare({
         routingKey:  '#'
       }
```

Comment:
Bindings was supported to be an array... but let's just give it a default value of `[]` in the JSON schema :)